### ISSUE TITLE
Implement a language switch so that users can choose to view the site in English, Hebrew, or Arabic 

### ISSUE THREAD

**Issue by gissoo** [Princeton-CDH/geniza#1113]
## testing notes (QA)

- [x] On the QA site, when logged in, confirm there is a language switcher with 3 languages (english, hebrew, arabic) matching the designs below
- [x] On desktop and mobile (if possible), confirm it switches the language of the current page
- [x] Navigate away and confirm the language change persists
- [x] Log out and confirm the switcher is still present, but only shows English and Hebrew
- [x] Confirm the language switcher options are properly aligned

## dev notes

@blms [here are the designs](https://www.figma.com/file/anZFStv7zBaVWnFxr95nKt/Work-in-Progress-%2F-WIP?node-id=2398%3A70147) – I'll eventually move them to the Components pages – I've provided select and hover designs for LTR and RTL pages for light and dark mode on desktop and mobile 

so on desktop and mobile the toggles will stay in the nav, for mobile this means that they are visible and usable once you've selected the burger menu 

You can see the designs on the pages if you zoom out of the Figma page – let me know if you need anything

**Comment by gissoo** [comment]
@blms I've applied the Semibold Greta Sans to "E" in the designs 
I also noticed the "עִ" wasn't centered in the toggle container, I have fixed that as well, fwiw. 

"E" is 20px on desktop and 18px on mobile (semibold weight Greta Sans for default, and Bold weight Greta Sans for selected)
"עִ" is 26px on desktop and 20px on mobile (regular weight Frank Ruehl for both default and selected)
"ع" is 20px on desktop and 16px on mobile. (bold weight Hassan for both default and selected)

**Comment by kseniaryzhova** [comment]
@gissoo the language symbol for Hebrew looks out of line with the English and Arabic 
![image](https://user-images.githubusercontent.com/105385128/192869659-a247fcc1-592e-411b-bff3-7bc525804b27.png)

**Comment by blms** [comment]
@kseniaryzhova Ooh, good catch. That looks like a browser difference, since I'm now seeing similar on Firefox but not Safari or Chrome.

**Comment by kseniaryzhova** [comment]
@gissoo couldn't test on mobile since I'm off campus, but everything else works on desktop! Should I close?

**Comment by blms** [comment]
@kseniaryzhova Please leave open and I'll fix the alignment issue you raised. (Also I should mention, the Arabic font is a placeholder, still waiting for the Hassan font from @gissoo but should be ready soon.)

**Comment by kseniaryzhova** [comment]
@gissoo I have the same alignment issue on Chrome too.

**Comment by kseniaryzhova** [comment]
@gissoo @blms alignment issue still exists for me - Arabic is too small (but I know the font isn't ready yet) but the Hebrew is again out of alignment with the English E. 
![image](https://user-images.githubusercontent.com/105385128/194071635-d5d0ae64-9f91-4650-bcbe-b3a9bd285c77.png)

**Comment by blms** [comment]
@kseniaryzhova Interesting! I suspected this might be a Windows vs Mac thing, and it is indeed. It has to do with the font's "vertical metrics". 

Here it is on both Windows and Mac Chrome with the margins I set highlighted in orange. Notice how in the Windows screenshot, the distance of the letter from the orange margin is smaller than it is on Mac. So basically on Windows this font has some extra space on top of each character!
![Untitled](https://user-images.githubusercontent.com/7234006/194109310-6c40522c-9c17-475f-9d51-39ab4306face.png)
![Screen Shot 2022-10-05 at 12 02 26 PM](https://user-images.githubusercontent.com/7234006/194109321-5797d8e5-4255-4d10-82a1-0c5fae761ca6.png)

Going to try using [this tool](https://transfonter.org/) (which I found in a StackOverflow thread) to edit the fonts and see if that fixes it.
